packages/react-components/src/unstable/tsconfig.json

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
this file is not needed, `unstable` is already in check path of `tsconfig.lib.json`